pkg/antctl/command_definition.go

```diff
@@ -94,6 +95,11 @@ var groupCommands = map[commandGroup]*cobra.Command{
 		Short: "Sub-commands of multi-cluster feature",
 		Long:  "Sub-commands of multi-cluster feature",
 	},
+	upgrade: {
+		Use:   "upgrade",
+		Short: "Execute a user-provided upgrade",
+		Long:  "Execute a user-provided upgrade",
```

Comment:
This is not clear to understand, maybe:
```suggestion
		Short: "Upgrade version of Kubernetes resources",
		Long:  "Upgrade version of Kubernetes resources",
```
@antoninbas @tnqn any suggestion?

Reply:
How about:
"Upgrade Antrea CRD resources"
"Upgrade Antrea CRD resources to the latest API version"

Reply:
I thought of that `upgrade` will be a group of commands. We could upgrade not only CRD resources, maybe we could upgrade something else used by Antrea. Do we need to limit the command `upgrade` to K8s resources or CRD right now?